proxy.go

```diff
@@ -177,6 +179,44 @@ func printAgentLogs(sock string) error {
 	return nil
 }
 
+func setupSigtermNotifier() chan os.Signal {
+	sigCh := make(chan os.Signal)
+	signal.Notify(sigCh, syscall.SIGTERM)
+
+	return sigCh
+}
+
+func handleSigtermSignal(sigCh chan os.Signal, vmConn net.Conn, proxyListener net.Listener) error {
```

Comment:
Again, I appreciate this is absolutely correct, but maybe `handleSignal()` would make it easier to read.

Reply:
Sure !